### ISSUE TITLE
Oxyacetylene has damage block in UI despite not being used for shooting

### ISSUE THREAD

**Issue by olanti-p** [cataclysmbnteam/Cataclysm-BN#2410]
### Describe the bug

Oxyacetylene has damage block in UI despite not being used for shooting or in guns.

### Steps To Reproduce

1. Spawn acetylene torch
2. See 'Ammunition' block with damage stats

### Screenshots

![image](https://user-images.githubusercontent.com/60584843/224575731-d1d05255-7797-44a2-a1ba-46ac7f9c2bd4.png)

### Versions and configuration

Game version ed644926

**Comment by chaosvolt** [comment]
Default behavior I suspect, does it do this with thread, batteries, and charcoal?

**Comment by olanti-p** [comment]
Yeah, sewing kit & charcoal smoker both show same ammo block. I'll edit the issue.
